affordable-housing/housing.html

```diff
@@ -0,0 +1,901 @@
+<!DOCTYPE html>

```

Comment:
This file should go into a separate pull request to the gh-pages branch. Please put it in a new directory here and link to it from index.html:

https://github.com/codeforsanjose/HackNightProjects/tree/gh-pages